src/components/ContentHighlights/data/reducer.js

```diff
@@ -0,0 +1,27 @@
+/*eslint-disable*/ 
```

Comment:
[curious] what rule(s) were causing an issue?

Reply:
Just an unending series of 'expected x indents found y'